cmd_vm.go

```
@@ -0,0 +1,429 @@
+		
+		return nil
+	})
+	
+	if err != nil {
+		GlobalLogger.Warning.Printf("Could not read disk information: %v\n", err)
+	}
+	
+	// Snapshot information
+	fmt.Println("\nSnapshots:")
+	snapshotCount := 0
+	err = filepath.Walk(vmBundle.Path, func(path string, info os.FileInfo, err error) error {
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)